src/test/java/com/networknt/schema/Issue366FailFast.java

```diff
@@ -35,7 +35,7 @@ private void setupSchema() throws IOException {
 
     URI uri = getSchema();
 
-    InputStream in = getClass().getResourceAsStream("/draft7/issue366_schema.json");
+    InputStream in = getClass().getResourceAsStream("/schema/issue366_schema.json");
```

Comment:
Perplexed why this test didn't fail for me when I ran it from "mvn clean test" or right-click test folder in IntelliJ and run all tests.   Shouldn't I have seen FileNotFoundException?

Reply:
getResourceAsStream seems to return null rather than throw an exception.

Reply:
yeah, weird I've just noticed when running mvn that it shows
```
[INFO] Running com.networknt.schema.Issue342Test
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in com.networknt.schema.Issue342Test
[INFO] Running com.networknt.schema.Issue347Test
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in com.networknt.schema.Issue347Test
[INFO] Running com.networknt.schema.Issue375Test
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.001 s - in com.networknt.schema.Issue375Test
```
so it's bypassing that test??

Reply:
If I rename it to Issue366FailFastTest.java, then mvn runs it.